### PR TITLE
Update support for AWS MediaConvert:

### PR DESCRIPTION
**Pull request by jdrossl**
- Migrate service to API2
- Improve service to return a list of urls for all configured outputs
- Use remote assets urls

https://github.com/craftercms/craftercms/issues/2959

### CODE REVIEW

File: src/main/api/2/studio-api-2.yaml [hunk 1]
```diff
@@ -1088,6 +1088,55 @@ paths:
...
+                file:
+                  type: string
+                  format: binary
+                  description: The content of the file to upload
+              required:
+                - siteId
+                - inputProfileId
+                - outputProfileId
+                - file
+      responses:
```
Comment (sumerjabri) [inline src/main/api/2/studio-api-2.yaml:1124]: No 404? What if the job profiles are not present?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/api/2/studio-api-2.yaml [hunk 2]
```diff
@@ -1088,6 +1088,55 @@ paths:
...
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  item:
+                    $ref: '#/components/schemas/MediaConvertResult'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
```
Comment (avasquez614) [inline src/main/api/2/studio-api-2.yaml:1142]: Could there be a bad request response if the request body has required or missing fields?

**Review by avasquez614 — CHANGES_REQUESTED** [review]

File: src/main/api/2/studio-api-2.yaml (hunk 1 above)
Comment (jdrossl) [reply src/main/api/2/studio-api-2.yaml:1124]: Any error from AWS related services will be mapped to a 500 response

File: src/main/api/2/studio-api-2.yaml (hunk 2 above)
Comment (jdrossl) [reply src/main/api/2/studio-api-2.yaml:1142]: The service will return a 400 response, I'll add it to the doc

**Review by avasquez614 — APPROVED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**